homeworks/nikolay.krasniuk_Kingstar22/6-typescript/src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 ;
+  padding: 0;
+  height: 100vh;
+  background: #C4D9FF;
+}
+
+.game {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  height: 60%;
+}
+
+.game__caunter {
+  padding: 10px;
+  background-color: #FFF;
+  border-radius: 10px;
+}
+
+.game__caunter-main {
+  margin: 0;
+  font-size: 40px;
+}
+
+.game__content {
+  display: flex;
+  justify-content: space-around;
+  width: 50%;
+  align-items: center;
+}
+
+.game__content-caunter {
+  width: 45px;
+  height: 45px;
+  padding: 10px;
+  background-color: #FFF;
+  border-radius: 10px;
+  margin: 0;
+  font-size: 20px;
+}
+
+.game__content-circle {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  background: radial-gradient(#FFF, #FFA9A1)
+}
+
+.game__footer-characters {
+  width: 70px;
+  height: 76px;
+  margin: 0;
+  padding: 10px;
+  background-color: #FFF;
+  border-radius: 10px;
+  font-size: 50px;
+  font-weight: 900;
+  text-align: center;
+}
+
+.game__footer-loader {
+  width: 70px;
+  height: 15px;
+  border-radius: 20px;
+  background: linear-gradient(rgb(245, 172, 36) 0 0) left/0% 100% no-repeat rgb(173, 216, 230);
+  /* animation: p2 2s infinite steps(10); */
```

Comment:
If you don't need commented code, it's better to remove it

Reply:
done